_posts/2022-03-25-text-gen-via-lfd.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: post
+title: An Understanding of Learning from Demonstrations for Neural Text Generation
+authors: Kantharaju, Pavan, Smart Information Flow Technologies; Sankar, Aiswarya, Independent
+tags: [Text Generation, Reinforcement Learning, Learning from Demonstration]  # This should be the relevant areas related to your blog post
+---
+
+# An Understanding of Learning from Demonstrations for Neural Text Generation 
```

Comment:
Could you remove this title as its a bit redundant:

![image](https://user-images.githubusercontent.com/19653980/163041260-6bfb6c5e-ee7d-4653-81ef-a675ddb1ce1a.png)

Reply:
Done!